api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/LocationBundle.java

```diff
@@ -6,7 +6,7 @@
 import org.springframework.stereotype.Component;
 
 @Component
-@Requires({ LocationTagBundle.class, LocationAttributeTypeBundle.class} )
+@Requires({ LocationAttributeTypeBundle.class} )
```

Comment:
I saw you added the attribute types to Iniz in config-pihemr... is there a reason you are still installing them here?

Reply:
Oversight. Thanks!